server/src/main/java/se2/ticktackbumm/server/network/ServerMessageHandler.java

```diff
@@ -95,4 +95,8 @@ public void handleSpinWheelFinished(GameMode gameMode) {
         serverData.getGameData().setCurrentGameMode(gameMode);
         serverMessageSender.sendGameUpdate();
     }
+
+    public void handleCardOpened(String word) {
+        serverMessageSender.sendCardOpened(word);
```

Comment:
Hier sollte die GameData am Server aktualisiert werden und die Veränderung den Spielern mittels GameUpdate mitgeteilt werden.

Reply:
Das `word` muss nicht in CardOpened mitgeschickt werden, das wird von `sendGameUpdate()` erledigt.